models/categoryModel.js

```diff
@@ -33,6 +33,10 @@ const CategorySchema = new Schema({
     type: Boolean,
     default: true,
   },
+  isFeatured: {
+    type: Boolean,
+    default: true,
```

Comment:
This does need to be false. Not every category will be featured. That will be at the discretion of the admin.

Reply:
changed isFeatured to false